.github/workflows/diff.yml

```diff
@@ -14,15 +14,25 @@ jobs:
     runs-on: ubuntu-latest
     container: obolibrary/odklite:v1.3.0
     steps:
```

Comment:
Given the total runtime, there should be check to ensure of the branch is up to date with master, and fail if it is not (or better yet, post a comment saying its failed because its not up to date).